packages/devtools_app/lib/src/screens/debugger/debugger_screen.dart

```diff
@@ -360,27 +360,29 @@ class DebuggerStatus extends StatefulWidget {
 class _DebuggerStatusState extends State<DebuggerStatus> with AutoDisposeMixin {
   String _status = '';
 
+  bool get _isPaused =>
+      serviceManager.isolateManager.mainIsolateState?.isPaused.value ?? false;
+
```

Comment:
perhaps we should add a utility method for this that can be shared. `isConnectedAppPaused` or similar in utils.dart

Reply:
serviceManager seems to be a proper place